frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunTabDetails.tsx

```diff
@@ -42,15 +48,15 @@ const PipelineRunTabDetails: React.FC<PipelineRunTabDetailsProps> = ({
           // TODO: get the relative parent namespaced link
           value: (
             <Link to={`/pipelines/${namespace}/pipeline/view/${pipelineReference.key.id}`}>
-              {pipelineReference.name}
+              <Truncate content={pipelineReference.name ?? ''} />
```

Comment:
Likely best to avoid rendering the link altogether if there's no `pipelineReference.name`, otherwise we just get an empty node if there's no text content.